config/plugin/check_ntp.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# NOTE: THIS NTP SERVICE CHECK SCRIPT ASSUME THAT NTP SERVICE IS RUNNING UNDER SYSTEMD.
+#       THIS IS JUST AN EXAMPLE. YOU CAN WRITE YOUR OWN NODE PROBLEM PLUGIN ON DEMAND.
+
```

Comment:
Check whether this node is using systemd? If not, return `Unknown`?

Reply:
Good catch. Will do.

Reply:
Actually what we use in our production env has those checks. :)